server/controllers/admin.py

```diff
@@ -440,7 +467,7 @@ def templates(cid, aid):
         if files:
             templates = {}
             for template in files:
-                templates[template.filename] = str(template.read(), 'latin1')
+                templates[template.filename] = str(template.read(), 'utf-8')
```

Comment:
Does it prevent uploads of zip files or images for example? (Scheme Art is a use case for submitting pictures)